Admin/CrossTenantMailboxMigrationValidation.ps1

```diff
@@ -1041,6 +1131,9 @@ if ($SourceIsOffline -and $PathForCollectedData -and $CheckOrgs) {
     Write-Verbose -Message "SourceTenantId gathered from SourceTenantId.txt: $SourceTenantId"
     $TargetTenantId = (Get-AzureADTenantDetail).ObjectId
     Write-Verbose -Message "TargetTenantId gathered from AzureADTenantDetail.ObjectId: $TargetTenantId"
+    $SourceAADApp = (Import-Clixml $OutputPath\SourceAADServicePrincipals.xml)
+    Write-Verbose -Message "Informational: Checking if there's already an AAD Application on TARGET tenant that meets the criteria"
+    $TargetAADApp = Get-AzureADApplication | Where-Object { ($_.ReplyUrls -eq "https://office.com") -and ($_.RequiredResourceAccess -like "*ResourceAppId: 00000002-0000-0ff1-ce00-000000000000*") }
```

Comment:
`Get-AzureADApplication` and other cmdlets which are part of the `AzureAD` module should be replaced in a new PR as the `AzureAD` module will be deprecated (see: https://learn.microsoft.com/en-us/powershell/module/azuread/?view=azureadps-2.0). 

We have some helper functions in place that utilize `Graph API` instead of `AzureAD` module. We introduced them with this PR:  #1638